docker/bitcoin-regtest/entrypoint.sh

```diff
@@ -5,10 +5,16 @@ set -e
 cd /opt/coins/nodes/bitcoin_regtest
 
 echo "Starting bitcoin regtest backend service"
-/opt/coins/nodes/bitcoin_regtest/bin/bitcoind -datadir=/opt/coins/data/bitcoin_regtest/backend -conf=/opt/coins/nodes/bitcoin_regtest/bitcoin_regtest.conf &
+bin/bitcoind -datadir=/opt/coins/data/bitcoin_regtest/backend -conf=/opt/coins/nodes/bitcoin_regtest/bitcoin_regtest.conf &
+
+sleep 3
+
+# generate test wallet and blocks
+bin/bitcoin-cli -rpcport=18021 -rpcuser=rpc -rpcpassword=rpc createwallet "tenv-wallet"
+bin/bitcoin-cli -rpcport=18021 -rpcuser=rpc -rpcpassword=rpc -generate 1024 1000
 
 # start blockbook
 cd /opt/coins/blockbook/bitcoin_regtest
 
 echo "Starting blockbook service"
-/opt/coins/blockbook/bitcoin_regtest/bin/blockbook -blockchaincfg=/opt/coins/blockbook/bitcoin_regtest/config/blockchaincfg.json -datadir=/opt/coins/data/bitcoin_regtest/blockbook/db -sync -internal=:19021 -public=:19121 -certfile=/opt/coins/blockbook/bitcoin_regtest/cert/blockbook -explorer= -log_dir=/opt/coins/blockbook/bitcoin_regtest/logs
+bin/blockbook -blockchaincfg=/opt/coins/blockbook/bitcoin_regtest/config/blockchaincfg.json -datadir=/opt/coins/data/bitcoin_regtest/blockbook/db -sync -internal=:19021 -public=:19121 -certfile=/opt/coins/blockbook/bitcoin_regtest/cert/blockbook -explorer= -log_dir=/opt/coins/blockbook/bitcoin_regtest/logs
```

Comment:
In this command, I was able to replace the absolute path with relative - deleting all the `/opt/coins/blockbook/bitcoin_regtest` prefixes of locations